src/_tkagg.cpp

```diff
@@ -100,39 +88,33 @@ static int PyAggImagePhoto(ClientData clientdata, Tcl_Interp *interp, int
     }
 
     /* check for bbox/blitting */
-    if (sscanf(argv[4], SIZE_T_FORMAT, &bboxl) != 1) {
-        TCL_APPEND_RESULT(interp, "error casting pointer", (char *)NULL);
-        return TCL_ERROR;
-    }
-    bboxo = (PyObject *)bboxl;
-
-    if (bboxo != NULL && bboxo != Py_None) {
-        agg::rect_d rect;
-        if (!convert_rect(bboxo, &rect)) {
-            return TCL_ERROR;
-        }
-
+    if (sscanf(argv[4], BBOX_FORMAT, &x1, &x2, &y1, &y2) == 4) {
```

Comment:
should probably error out cleanly if an invalid string is passed?

Reply:
yesish, these are basically "private" functions parsing a call from python, but I will add more checking code

Reply:
updated in cc7b73e